crates/symbolicator/src/services/symbolication.rs

```diff
@@ -1316,6 +1312,23 @@ fn symbolicate_stacktrace(
                     continue;
                 }
 
+                // Certain linux compilers/libc combinations create a `DW_CFA_undefined: RIP` DWARF
+                // rule to say that `_start` has no return address. We do not support this and will
+                // thus use the previous rule for RIP, which might look up the register value on the
+                // stack, resulting in an unmapped garbage frame. We work around this by trimming the
+                // trailing garbage frame on the following conditions:
+                // * this is the last frame (via peek)
+                // * it is unmapped (UnknownImage)
+                // * the previous frame is `_start`
+                let is_start =
+                    |frame: &SymbolicatedFrame| frame.raw.function.as_deref() == Some("_start");
+                if status == FrameStatus::UnknownImage
+                    && frames_iter.peek().is_none()
+                    && frames.last().map_or(false, is_start)
+                {
+                    continue;
```

Comment:
do you suppose it'd be useful to eventually track these garbage frames (and other skipped frames) with metrics or logging eventually in the future?

Reply:
I don’t think this makes particular sense. We had a discussion recently about measuring with intent, and not just throw a metric on every little thing. I’m not sure how such a metric can drive us in a direction to improve this or not.